content/numerical/FastFourierTransform.h

```diff
@@ -27,25 +36,19 @@ void fft(vector<C> &a, vector<C> &rt, vi& rev, int n) {
 			C z(x[0]*y[0] - x[1]*y[1], x[0]*y[1] + x[1]*y[0]);           /// exclude-line
 			a[i + j + k] = a[i + j] - z;
 			a[i + j] += z;
-	}
+		}
 }
-
-vd conv(const vd& a, const vd& b) {
+vd conv(const vd &a, const vd &b) {
```

Comment:
KACTL is relatively consistent about using `const vd& a`; let's stick with that (same in convMod)